lib/cli.rb

```diff
@@ -85,7 +86,7 @@ def version_option(parser)
     class TestOptions
       attr_accessor :platform, :drivers, :driver_path, :commit, :diff_file, :svvp, :manual,
                     :gthb_context_prefix, :gthb_context_suffix, :playlist, :select_test_names,
-                    :reject_test_names
+                    :reject_test_names, :reject_report_section
```

Comment:
The option is named `--reject-report-sections` and its value is an `Array`, but this attribute is not plural. It is confusing.